### PR TITLE
Add HOCON lexer.

### PR DESCRIPTION
**Pull request by davidtwco**
This PR adds a lexer for [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md). I'm not sure that exhaustively covers all of HOCON, but I believe it covers the majority of the language.

I'm happy to make any modifications, improvements or changes as requested. Thanks.

### CODE REVIEW

File: lib/rouge/lexers/hocon.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*- #
+
```
Comment (vidarh) [inline lib/rouge/lexers/hocon.rb]: Can you add '# frozen_string_literal: true'

File: spec/lexers/hocon_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*- #
+
```
Comment (vidarh) [inline spec/lexers/hocon_spec.rb]: Can you add '# frozen_string_literal: true'

File: lib/rouge/lexers/hocon.rb (hunk 1 above)
Comment (davidtwco) [reply lib/rouge/lexers/hocon.rb]: Fixed.

File: spec/lexers/hocon_spec.rb (hunk 2 above)
Comment (davidtwco) [reply spec/lexers/hocon_spec.rb]: Fixed.

**Comment by davidtwco** [comment]
This is blocked on #1062 to get the CI to pass.

**Comment by davidtwco** [comment]
Closing this as it appears to be being handled in pyrmont#51.

**Comment by ashmaroli** [comment]
@davidtwco Hi. The PR you linked to was a workaround when this project got stalled temporarily.
However, Rouge is back in active development. Therefore, if you would like this lexer to be shipped eventually, you're welcome to re-open and update the branch in sync with latest `master` and fix any failing tests.
Thanks.

**Comment by davidtwco** [comment]
@ashmaroli I've submitted #1253.

**Pull request closed without merging**